lib/boxxspring/response.rb

```diff
@@ -38,17 +37,13 @@ def failure?
     end
     
     protected; def decode_response_body( http_response )
-    
-      response = nil
-      
-      body = http_response.body;
-      unless body.nil? || body.empty?
-        response = 
-          JSON.parse( body ) rescue nil
+      body = http_response.body
+
+      if body.nil? || body.empty?
+        nil
+      else
+        JSON.parse( body ) rescue nil
```

Comment:
well if you're tweaking these lines, might i suggest just doing.
```if body.present? 
JSON.parse( body ) rescue nil
else
nil 
end```
 essentially the same but I like the positive coming first